apps/omg_watcher_informational/lib/omg_watcher_informational/event_consumer.ex

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInformational.EventConsumer do
```

Comment:
naming here? we have `DepositConsumer` and `EventConsumer`, the latter doing only `ExitStarted`. So need to decide whether we have 1 consumer for all or split